uber/sep_commands.py

```diff
@@ -70,7 +157,13 @@ def insert_admin():
 
 
 @entry_point
+def drop_uber_db():
+    assert c.DEV_BOX, 'drop_uber_db is only available on development boxes'
+    Session.initialize_db(modify_tables=False, drop=True)
+
+
+@entry_point
 def reset_uber_db():
     assert c.DEV_BOX, 'reset_uber_db is only available on development boxes'
-    Session.initialize_db(drop=True, modify_tables=True)
+    Session.initialize_db(modify_tables=True, drop=True)
```

Comment:
Hmm, I was going to ask whether this is even something we still want to support, since in theory we could call ``alembic('upgrade', 'heads')`` here and even that would be redundant with just saying ``sep alembic upgrade heads``.  But I see that you're using ``reset_uber_db()`` below to test the schema migrations, so ignore me
(https://www.youtube.com/watch?v=FMNJuSl91qY&t=7s).